book/src/filters.md

```diff
@@ -233,6 +233,34 @@ Output:
 HELLO
 ```
 
+### urlencode
+
+Percent-encodes the value for safe use in URI; does not encode `/`.
```

Comment:
The example here does encode the slash. Which is correct?

Reply:
I just it from the docs, and rendered the examples.

I'll look into whether `urlencode` has always been buggy, or if some upstream change caused it.

Reply:
It's probably due to https://github.com/djc/askama/pull/384.